airflow-core/src/airflow/api_fastapi/core_api/datamodels/ui/config.py

```diff
@@ -31,6 +31,7 @@ class ConfigResponse(BaseModel):
     enable_swagger_ui: bool
     require_confirmation_dag_change: bool
     default_wrap: bool
+    default_show_timestamp: bool
```

Comment:
@bbovenzi @pierrejeambrun I'm a little surprised to see in the server -- do purely UI concerns such as wrap and timestamp even make sense to have here? They aren't configurable via Airlfow config, so these values don't ever change do they?

Reply:
This was removed as per @bbovenzi 's suggestion. Resolved in e3f4eae.